app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    params['page'] = 1 if params['page'].nil? || ((params['page'].to_i < 1))
+    @user = User.find(params['user_id'].to_i)
+    @posts = @user.posts.limit(2).offset((params['page'].to_i - 1) * 2)
+    @page = params['page'].to_i
+  end
```

Comment:
- [ ] Great work implementing pagination for this application. I must confess, this is outstanding and unique. Keep up the great work. 👏